Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/FileTransporter.cs

```diff
@@ -120,13 +131,17 @@ public FileTransferResult Run(CancellationToken cancellationToken)
             catch (SocketException)
             {
                 userResultMessage = $"Unable to establish the socket: No such host is known.";
-                return FileTransporter.FormFailedFileTransferResult(userResultMessage);
+                _logger.NotifyError(_batchContext, userResultMessage, new Exception(userResultMessage));
+                return FormFailedFileTransferResult(userResultMessage);
             }
 
             return FormResultFromSingleTransferResults(Result);
         }
 
         #region Helper methods
+        // ExcludeFromCodeCoverage(Justification) is not supported for 4.7.1 and 2.0:
+        // This one is excluded because currently, it seems impossible to get past 79% coverage in CI, even though it's over 81% when run locally.
```

Comment:
This is not a valid reason for excluding method from coverage test. Since this is a static method you can write a unit test which tests this method to make sure it is covered.